lib/puppet/transaction.rb

```diff
@@ -322,7 +322,7 @@ def prefetch(provider_class, resources)
     begin
       provider_class.prefetch(resources)
     rescue LoadError, Puppet::MissingCommand => detail
-      Puppet.log_exception(detail, "Could not prefetch #{type_name} provider '#{provider_class.name}': #{detail}")
+      Puppet.log_exception(detail, _("Could not prefetch #{type_name} provider '#{provider_class.name}': #{detail}"))
```

Comment:
wow this is going to be a hard string to translate. 

`prefetch` is a specific function name / term in puppet and shouldn't be translated anywhere - but in this string it's not marked uniquely or separately from any other regular word to be translated. Is there some way we're supposed to notify the translator of these kinds of terms? Maybe a code comment that is pulled into the POT file? cc @LarissaLane